wp-content/plugins/wporg-learn/views/block-workshop-details.php

```diff
@@ -17,10 +17,31 @@
 <div class="wp-block-wporg-learn-workshop-details">
 	<?php if ( ! empty( $fields ) ) : ?>
 		<ul class="workshop-details-list">
-			<?php foreach ( $fields as $key => $value ) : ?>
+			<?php foreach ( $fields as $key => $field ) : ?>
 				<li>
-					<b><?php echo esc_html( $key ); ?></b>
-					<span><?php echo esc_html( $value ); ?></span>
+					<b><?php echo esc_html( $field['label'] ); ?></b>
+					<span>
+						<?php
+						$i = 0;
+						foreach ( $field['value'] as $field_key => $value ) {
+							$url = '';
+							if ( ! empty( $field['param'] ) ) {
+								$url = trailingslashit( site_url() ) . 'workshops/?' . $key . '=' . $field['param'][ $field_key ];
+							}
+
+							if ( 0 < $i ) {
+								echo ', ';
+							}
+
+							if ( $url ) {
+								echo sprintf( '%1$s' . esc_url( $url ) . '%2$s' . esc_html( $value ) . '%3$s', '<a href="', '">', '</a>' );
```

Comment:
What's the reason for using sprintf() in this way? (also a similar line added in component-taxonomy-item.php)

Reply:
A naked echo felt...weird to do for some reason, so honestly I don't have a better reason than that.